dandi/core/digests/dandietag.py

```diff
@@ -0,0 +1,103 @@
+from hashlib import md5
+import math
+import os
+
+# from https://github.com/girder/django-s3-file-field/blob/master/s3_file_field/_multipart.py
+from typing import List, Optional, Tuple
+
+
+def mb(bytes_size: int) -> int:
+    return bytes_size * 2 ** 20
+
+
+def gb(bytes_size: int) -> int:
+    return bytes_size * 2 ** 30
+
+
+def tb(bytes_size: int) -> int:
+    return bytes_size * 2 ** 40
+
+
+class DANDIEtag:
```

Comment:
```suggestion
class DandiETag:
```

This is how we've been naming classes in dandi-api (`DandiSettings`, `DandiPagination`, `DandisetViewSet`, etc). I think treating "Dandi" were a proper noun makes it much more wieldy, even if it is technically an acronym.

Reply:
right -- and we have `DandiAPIClient` so let's make it `DandiETag` here.